packages/client/src/components/shared/ExtendButton/ChangeableButton.component.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FaChevronRight } from 'react-icons/fa';
+import './ChangeableButton.styles.css';
+
+export const ChangeableButton = ({ text, onClick }) => {
```

Comment:
Took me a minute to understand what this button is! 
Since it is such a unique component by its functionality, I think it could even be called ReadMoreButton - then it is immediately clear what it is.

Reply:
Good point, thanks.